test/infra/cchost.py

```diff
@@ -182,27 +182,22 @@ async def _start_process(self) -> None:
             cchost_env["FAKETIME_NO_CACHE"] = "1"
 
         LOG.debug("Starting cchost process...")
+        stdout_file = open(f"{self.workspace}/std.out", "w")
+        stderr_file = open(f"{self.workspace}/std.err", "w")
```

Comment:
Could it make sense to print the content of these files when the cchost process terminates to improve the debugging of possible cchost errors? If we just write them into files, it's hardly useful unless someone debugs the tests locally and check for these files explicitly

Reply:
The idea is to upload them for later inspection, with upload-artifact in GHA or equivalent. Then you can download them all if you need to debug something.

Reply:
There are being uploaded afterwards

Reply:
Same in GH pipelines, but on failure only